python3/splashkit.py

```diff
@@ -1,19 +1,27 @@
 from ctypes import *
 from enum import Enum
 from platform import system
+import os
```

Comment:
Are the changes in this file coming through the translator?

Reply:
Yes, the translator change has been merged by you in [this PR](https://github.com/splashkit/splashkit-translator/pull/60) :)